### PR TITLE
Taz/add nav bar

### PR DESCRIPTION
**Pull request by Taz-Z**

### CODE REVIEW

File: Project/src/main/resources/templates/upload.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 </head>
 <body>
+<header th:insert="header.html :: header"></header>
```
Comment (DanGravel) [inline Project/src/main/resources/templates/upload.html]: tab

File: Project/target/classes/templates/default.html [hunk 2]
```diff
@@ -3,8 +3,10 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 </head>
 <body>
+<header th:insert="header.html :: header"></header>
```
Comment (DanGravel) [inline Project/target/classes/templates/default.html:9]: tab

File: Project/src/main/resources/templates/reviewer.html [hunk 3]
```diff
@@ -3,8 +3,10 @@
 <head>
     <meta charset="UTF-8">
     <title>Reviewer</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 </head>
 <body>
+<header th:insert="header.html :: header"></header>
```
Comment (DanGravel) [inline Project/src/main/resources/templates/reviewer.html]: tab

File: Project/src/main/resources/templates/default.html [hunk 4]
```diff
@@ -3,8 +3,10 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 </head>
 <body>
+<header th:insert="header.html :: header"></header>
```
Comment (DanGravel) [inline Project/src/main/resources/templates/default.html]: tab

**Review by DanGravel — CHANGES_REQUESTED** [review]

File: Project/src/main/resources/templates/editor.html [hunk 5]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
 <html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
 <head>
-    <meta charset="UTF-8">
-    <title>Dashboard</title>
```
Comment (acournoyer1) [inline Project/src/main/resources/templates/editor.html:5]: Why are these lines being removed?

Comment (Taz-Z) [reply Project/src/main/resources/templates/editor.html:5]: Part of another PR where theyre being added w/ different names

**Review by DanGravel — APPROVED** [review]

**Review by acournoyer1 — APPROVED** [review]

**Review by liamhickey — APPROVED** [review]
Good work

**Pull request merged**